conf/messages

```diff
@@ -5,7 +5,7 @@ nonProductionBanner.tag = NON-PRODUCTION
 nonProductionBanner.content = This is a non-Production environment, DO NOT use it to check real bank account information!
 
 notSecureBanner.tag = WARNING
-notSecureBanner.content = This BARS front-end tool will soon require a Stride login to use. Ask your line manager to grant you the "BARS Front End Tool User" SRS role to access the logged-in version of this page.
+notSecureBanner.content = We have improved security on this tool by adding authentication. Soon you will require to login using your stride credentials. Ask your line manager to grant you the SRS role. Please follow our guidance for next steps <a href="__AUTH_GUIDE_URL__">here</a>.
```

Comment:
Should this say 'will be required' or 'soon we will require login'?

Reply:
Good question - let me check the ticket.

Reply:
It looks like that is correct- according to CIR-1420